Example_DynamicFramework/PointziDemo/AppDelegate.m

```diff
@@ -16,6 +16,7 @@
  */
 
 #import "StreetHawkCore_Pointzi.h"
+#import "PZApp.h" //temp, when merge PZApp.h into StreetHawkCore_Pointzi.h, remove this line
```

Comment:
After https://bitbucket.org/shawk/pointzi-sdk-ios/pull-requests/148/add-missing-header-files-to-umbrella/diff merge, this temp line can be removed.